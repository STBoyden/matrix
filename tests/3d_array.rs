// Test that compiler will error if a > 2d array is passed as an argument
use matrix_multiplication::matrix;

fn main() {
    let _x = matrix!([[1, 2, 3], [4, [5], 6], [7, 8, 9]]);
}
