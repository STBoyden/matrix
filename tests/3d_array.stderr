error: Expected a 2d array
 --> $DIR/3d_array.rs:5:38
  |
5 |     let _x = matrix!([[1, 2, 3], [4, [5], 6], [7, 8, 9]]);
  |                                      ^^^
